ipaserver/install/installutils.py

```diff
@@ -893,9 +893,12 @@ def load_pkcs12(cert_files, key_password, key_nickname, ca_cert_files,
                 "The full certificate chain is not present in %s" %
                 (", ".join(cert_files)))
 
-        for nickname in trust_chain[1:]:
+        # verify CA validity and pathlen. The trust_chain list is in reverse
+        # order. The first entry is the signed IPA-CA and must have a
```

Comment:
The comment is wrong - this is the method for checking 3rd party service certs right?
Should say "The first entry is the end entity certificate..."

Reply:
The list is in reverse order (``trust_chain = list(reversed(nssdb.get_trust_chain(key_nickname)))``) and starts with the first intermediate certificate.

Reply:
I have updated the comment.

Reply:
Great, thanks!